README.md

```diff
@@ -251,8 +253,7 @@ pact.verifyPacts({
 | Parameter                   | Required? | Type    | Description                                                                                                |
 | --------------------------- | --------- | ------- | ---------------------------------------------------------------------------------------------------------- |
 | `providerBaseUrl`           | true      | string  | Running API provider host endpoint.                                                                        |
-| `pactBrokerBaseUrl`         | false     | string  | Base URL of the Pact Broker from which to retrieve the pacts.                                                                        |
-| `pactBrokerUrl`             | false     | string  | URL of your Pact Broker to dynamically discover relevent pacts to verify. Required if `pactUrls` not given |
```

Comment:
By removing this, we're breaking the API, hence a new version is needed.  Even with a message that's saying it's deprecated, it's still extra work for a developer to change the behaviour of their code.  I say remove all references to it everywhere, keep the deprecation message though, and increase the version to 8.0.0.